src/components/bulkorder.main.less

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+@import './../style/common.less';
+
+.hideModal{
+  display: none;
+}
+
+.bulk-order-component {
+  position: absolute;
+  top: 100%;
+  right: 0;
+  width: 460px;
+  border-bottom: 0;
+
+  .bulk-order-close-button{
+    background: #ffffff url(../images/header-icons/baseline_close_black_22dp.png) no-repeat 0px;
+    width: 70px;
+    height: 25px;
+    position: absolute;
+    top: 10px;
+    right: 10px;
+    z-index: 20;
+    cursor:pointer;
+
+    .bulk-order-hide{
+      padding-left: 25px;
+      padding-top: 3px;
+      font-size: 16px;
+      color: #333333;
+    }
+  }
+
+  .bulk-modal {
+    position: relative;
```

Comment:
Feedback from Bonnie is to have the panel cover the entire page:

![image](https://user-images.githubusercontent.com/39800563/55905485-064b3d80-5ba0-11e9-8dfa-ad496fac3e60.png)